engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-traces-modal.html

```diff
@@ -0,0 +1,75 @@
+<div data-extend-template="components/views/bst-modal.html">
+  <h4 data-block="modal-header">Content Host Traces Management</h4>
+
+  <div data-block="modal-body">
+    <form id="traceActionForm" name="traceActionForm" class="form" method="post" action="/katello/remote_execution">
+      <input type="hidden" name="remote_action" value="service_restart"/>
+      <input type="hidden" name="name" ng-value="traceActionFormValues.service"/>
+      <input type="hidden" name="host_ids" ng-value="traceActionFormValues.hostIds"/>
+      <input type="hidden" name="customize" ng-value="traceActionFormValues.customize"/>
+      <input type="hidden" name="authenticity_token" ng-value="traceActionFormValues.authenticityToken"/>
+    </form>
+
+    <div class="row">
+      <div class="col-sm-12">
+        <div bst-global-notification></div>
+      </div>
+    </div>
+
+    <div data-extend-template="layouts/partials/table.html">
+      <div data-block="search" ng-show="tracesearch"></div>
```

Comment:
I guess we can just remove this <div> and the $scope.tracesearch since we're not using it. Can be added back should be come along to add that function!

Reply:
Or do we actually need this empty div in order to hide it? In which case we can just do (I think):

```
<div data-block="search" ng-show="false"></div>
```

Reply:
The `<div data-block="search" ng-show="false"></div>` worked to hide it still, so I will remove the `scope tracesearch` and reference to it